README.md

```diff
@@ -476,6 +485,8 @@ Then(/^the search field exists$/) do
 end
 ```
 
+The method delegates to [Capybara::Node::Matchers#has_selector?](https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#has_selector%3F-instance_method)
```

Comment:
Move this to line 462 (After the name) - Also rename the start to `This method`

Same for all other instances below